docker-compose.override.yaml

```diff
@@ -0,0 +1,17 @@
+version: '3'
+
+services:
+    composer:
+        volumes:
+            - ./composer.json:/app/composer.json
+            - ./composer.lock:/app/composer.lock
+            - ./symfony.lock:/app/symfony.lock
+            - vendor:/app/vendor
+    fpm:
+        volumes:
+            - ./:/var/www/html/
+            - /var/www/html/var
+            - vendor:/var/www/html/vendor
```

Comment:
Usual caveat that this will be shared between builds. Would be interested to know if others have solutions. /cc @elrossco22 @stephenwf

Reply:
Sorry @thewilkybarkid not sure I follow, shared between builds where?

Reply:
Been using this technique to 'unmount' subdirectories  (so `var` isn't pulled back to the host, and `vendor` comes from the `composer` image rather than the host (which could be empty, so would delete the contents)). The problem is that the volume is persisted, so changes might not be recognised on future builds. See also https://github.com/libero/walking-skeleton/pull/34.

If we're going Docker only, we might be able to stop these being subdirectories by having them elsewhere on the filesystem (means not using the standard Composer `vendor` directory etc).